.github/workflows/POAM.yml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Team Sync
+on:
+  schedule:
+    - cron: 0 * 1 1-12 *
```

Comment:
Would `0 0 1 * *` suffice here?

Source: https://crontab.guru/once-a-month

Reply:
Oh yeah that's great! Thanks. I used the same tool to set the current cron schedule and thought I needed to be more verbose. Swapped for solution in [c73e011](https://github.com/uswds/uswds-team/pull/366/commits/c73e011d845c4a50249faa52b653bb719f1b3d67)